apps/yapms/src/lib/types/Candidate.ts

```diff
@@ -3,6 +3,7 @@ import { z } from 'zod';
 export const CandidateSchema = z.object({
 	id: z.string(),
 	name: z.string(),
+	count: z.number().optional(),
```

Comment:
This property already contains the data for the default count. Maybe this could be named something else such as "defaultCount" or "permaCount" or something that represents it's not just the current count.

Maybe this could also be non-optional and just have it defaulted to zero. That would make the calculations for where this is used a bit easier.